beets/importer.py

```diff
@@ -943,7 +943,9 @@ def is_archive(cls, path):
         if not os.path.isfile(path):
             return False
 
-        for path_test, _ in cls.handlers():
+        for path_test, x in cls.handlers():
+            if isinstance(path, bytes):
+                path = util.py3_path(path)
```

Comment:
I definitely hedged on that  because I couldn't use pdb for these tests, and logging statements break the tests!.  I was hoping you could tell me.

It seems to first occur in the pretend import tests.  I can't see where I'm missing anything. I can tell you that it's track_2.mp3.